app/helpers/application_helper.rb

```diff
@@ -54,5 +54,9 @@ def show_last_interaction_date_and_result(patient)
 
     interaction_result
   end
+
+  def patient_has_street_address(patient_address)
```

Comment:
Wondering whether this necessitates a helper. If we keep it, would be valuable to make a boolean function.
```suggestion
  def patient_has_street_address?(patient_address)
```

Reply:
Yes, I did debate whether such a short function could be inlined. However, I feel like it would still be worth keeping it separate from the view (cleaner). I will make the predicate more explicit by adding the `?`.

Reply:
You could add it as an instance method on `PatientAddress` itself, might be cleaner.

```
patient_address.has_street_address?
```

Reply:
@timcheadle Yes, that does make more sense. Removed the helper method and added predicate to the `Address` model instead.